test/lib/jdk/test/lib/process/ProcessTools.java

```diff
@@ -562,7 +596,7 @@ public static OutputAnalyzer executeTestJvm(List<String> cmds) throws Exception
      * @return The output from the process.
      */
     public static OutputAnalyzer executeTestJvm(String... cmds) throws Exception {
-        ProcessBuilder pb = createTestJvm(cmds);
+        ProcessBuilder pb = createTestJavaProcessBuilder(cmds);
```

Comment:
This should also describe *all* of the options being set in the ProcessBuilder before executing the process.